src/validation/ValidationError.ts

```diff
@@ -45,14 +45,20 @@ export class ValidationError {
    * @param shouldDecorate decorate the message with ANSI formatter escape codes for better readability
    * @param hasParent true when the error is a child of an another one
    * @param parentPath path as string to the parent of this property
+   * @param useConstraintMessages true to use the constraint messages instead of constraint names
```

Comment:
```suggestion
   * @param showConstraintMessages show constraint messages instead of constraint names
```

Reply:
I think the word "use" is a bit unclear here.

Reply:
I'm fine with that 😄